proto/axelar/evm/v1beta1/events.proto

```diff
@@ -222,6 +222,14 @@ message ContractCallApproved {
       [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Hash" ];
 }
 
+message ContractCallFailed {
+  string chain = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  string msg_id = 2 [ (gogoproto.customname) = "MessageID" ];
+  string reason = 3;
```

Comment:
Best to avoid adding not well defined attributes to events. Error strings can change/aren't part of the API spec. And this is not being used anymore anyways

Reply:
removed